pym/bob/cmds/build.py

```diff
@@ -474,9 +474,12 @@ def saveBuildState(self):
 
     def loadBuildState(self):
         state = BobState().getBuildState()
-        self.__wasRun = dict(state['wasRun'])
```

Comment:
Better use `dict(state.get('wasRun', {}))`. That'll save the else clause...

Reply:
:+1: too easy..